<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>112</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>361</width>
    <height>112</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>112</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Asset ID Hasher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="pathLabel">
      <property name="text">
       <string>Path:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="pathLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="resultLabel">
      <property name="text">
       <string>Result:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="resultLineEdit">
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="addPrefixCheckbox">
        <property name="text">
         <string>Prepend $/</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="toLowerCheckbox">
        <property name="text">
         <string>Lowercase</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Algorithm:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>CRC32</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MD5</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pathLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>onTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toLowerCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onCheckStateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addPrefixCheckbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>onCheckStateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>onCheckStateChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onTextChanged(QString)</slot>
  <slot>onCheckStateChanged()</slot>
 </slots>
</ui>
